cmake/rapids_config.cmake

```diff
@@ -26,4 +28,8 @@ else()
 endif()
 
 set(rapids-cmake-version "${RAPIDS_VERSION_MAJOR_MINOR}")
+if(NOT _rapids_branch)
```

Comment:
Nit: Move line 15-15 here so the declaration and error checking can stay together for better code understanding.